### ISSUE TITLE
Rancher logs show repetitive iNFO message `cri-dockerd is enabled for cluster version [v1.24.2-rancher1-1]`

### ISSUE THREAD

**Issue by markusewalker** [rancher/rancher#38192]
**Rancher Server Setup**
- Rancher version: `v2.6-head` https://github.com/rancher/rancher/commit/069433410f08903ae5967e5178867c2622bce419
- Installation option (Docker install/Helm Chart): Helm Chart
   - If Helm Chart, Kubernetes Cluster and version (RKE1, RKE2, k3s, EKS, etc): RKE1
- Proxy/Cert Details:

**Information about the Cluster**
- Kubernetes version: `v1.24.2-rancher1-1`
- Cluster Type (Local/Downstream): Downstream
   - If downstream, what type of cluster? (Custom/Imported or specify provider for Hosted/Infrastructure Provider): Linode

**User Information**
- What is the role of the user logged in? (Admin/Cluster Owner/Cluster Member/Project Owner/Project Member/Custom)
  - If custom, define the set of permissions:

   
**Describe the bug**

When provisioning a downstream RKE1 1.24 cluster, the following INFO logs are seen in the Rancher logs:

```
2022/07/05 18:35:28 [INFO] cri-dockerd is enabled for cluster version [v1.24.2-rancher1-1]
2022/07/05 18:35:58 [INFO] cri-dockerd is enabled for cluster version [v1.24.2-rancher1-1]
2022/07/05 18:37:20 [INFO] cri-dockerd is enabled for cluster version [v1.24.2-rancher1-1]
2022/07/05 18:40:35 [INFO] cri-dockerd is enabled for cluster version [v1.24.2-rancher1-1]
2022/07/05 18:41:04 [INFO] cri-dockerd is enabled for cluster version [v1.24.2-rancher1-1]
.
.
.
```
We should reconfigure these logs to be in either debug or trace-level logging. Aside from the multiple occurrences of this messaging happening, it's not necessarily important for a user to know that `cri-dockerd` is enabled or not. If that's the case, then you could make the argument that we should do this for all other flags when provisioning a downstream cluster.

An alternative would be to just remove this message altogether.

**To Reproduce**
 
1. Setup Rancher and navigate to the UI in a browser.
2. Create a standard user and login to Rancher as that user.
3. Provision a downstream RKE1 cluster and ensure that `cri-dockerd` is enabled.
     - In `Cluster Options`, click `Edit as YAML` and change from `false` to `true`. Then create the cluster.
4. Once provisioned, check the Rancher logs; you should see the above repetitive logging.

**Result**
A repetitive message of `cri-dockerd` being enabled comes up as INFO.

**Expected Result**

You should not see this repetitive message, instead it should be in a more fine-grained level of logging or removed altogether.

**Comment by phillipsj** [comment]
The decision is to move the info message to debug.

**Comment by markusewalker** [comment]
Still reproducing this behavior in `v2.6-head` https://github.com/rancher/rancher/commit/c47178eb1457a7ff1aac135c86d3627dc971b704.

**ENVIRONMENT DETAILS**
- Rancher Install: Helm
- Rancher version: `v2.6-head` https://github.com/rancher/rancher/commit/c47178eb1457a7ff1aac135c86d3627dc971b704
- Browser: Chrome

**TEST RESULT**
FAIL

**REPRODUCTION STEPS**
1. Setup Rancher and navigated to the UI in a browser.
2. Created a standard user and logged into Rancher as that user.
3. Provisioned a downstream RKE1 AWS node driver cluster; verified it provisioned successfully.
4. As admin, navigated back to `local` cluster and viewed the Rancher logs.
5. I still am seeing the repetitive error messages in the INFO message:
![image](https://user-images.githubusercontent.com/52591819/180282658-c5c79424-aadf-40e6-aa0d-2a31fc446eb5.png)

@HarrisonWAffel I saw that the PR was addressed yesterday, so I tested it out. Is it possible that the change just isn't in the current commit yet? I'll push this back to reopened for now, but please feel free to move back to `to-test` if I just need to wait for the next commit.

**Comment by HarrisonWAffel** [comment]
@markusewalker I think we need a new RKE RC, nothing in rancher/rancher has been changed yet.

**Comment by markusewalker** [comment]
Cool, thanks for the insight! I'll retest this once we get a new RKE RC and the changes are applied.

**Comment by HarrisonWAffel** [comment]
RKE Bump: https://github.com/rancher/rancher/pull/38366

**Comment by markusewalker** [comment]
Verified that this is addressed on `v2.6-head` https://github.com/rancher/rancher/commit/e464ea4056775539e94c7238b95ef56efa46ecee.

**ENVIRONMENT DETAILS**
- Rancher Install: Helm
- Rancher version: `v2.6-head` https://github.com/rancher/rancher/commit/e464ea4056775539e94c7238b95ef56efa46ecee
- Browser: Chrome

**TEST RESULT**
PASS

**VERIFICATION STEPS**
1. Setup Rancher and navigated to the UI in a browser.
2. Created a standard user and logged into Rancher as that user.
3. Provisioned a downstream RKE1 AWS node driver cluster; verified it provisioned successfully.
4. As admin, navigated back to local cluster and viewed the Rancher logs.
5. Verified that I no longer see the repetitive messages in the INFO/WARNING level log level.
